src/components/ListItem.js

```diff
@@ -0,0 +1,24 @@
+import './ListItem.css';
+
+export default function ListItem({ curry, onDelete }) {
```

Comment:
When I read `ListItem` as a component name, I think it is _one_ list item and not a list. Maybe rename this to `ShoppingList`? And please get rid of `curry`.